README.md

```diff
@@ -1,3 +1,51 @@
+# Documentation Guidelines
+
+Welcome to the Docusaurus documentation repository. When contributing documentation, please adhere to the following standards to ensure consistency and clarity.
+
+## Document Header
+
+Each document should start with a header section that defines metadata used for SEO and navigation. Please follow the below structure:
+
+```yaml
+---
+title: Title of the Document
+description: A concise SEO-friendly description, limited to 160 characters.
+tags: [tag1, tag2, tag3]
+---
+```
+
+- **title**: The title of the document.
+- **description**: his is critical for SEO. Keep it under 160 characters to ensure full visibility in search engine results.
+- **tags**: Include relevant tags that describe the document's content. This helps in categorizing the documentation.
+
+## Linking Strategy
+
+When adding links to other documentation pages, use absolute path links starting from the root of the documentation directory.
```

Comment:
@erenkan IMO relative and absolute path sometimes mixing for that we can give a sample. like that:

- Location of the file to be linked in the repo: `./docs/build/build-process-management/build-profile-branch-operations.md`
- The path must be given: `/build/build-process-management/build-profile-branch-operations`

It is a "nice to" advice